8_reference/73_techniques_best_practices.txt

```diff
@@ -25,6 +25,18 @@
 
 ==== Writing convention
 
+===== Technique Naming Guidelines
```

Comment:
Just like your guidelines say for technique names, I think this should not be capitalised ;)